packages/lib-classifier/src/store/utils/ClassificationQueue.spec.js

```diff
@@ -109,14 +109,14 @@ describe('ClassificationQueue', function () {
   })
 
   describe('keeps classifications in localStorage if backend fails', function () {
-    let clock
-
     before(function () {
-      clock = sinon.useFakeTimers({ global })
+      sinon.stub(global, 'setTimeout').callsFake(() => 100)
```

Comment:
You've changed the return type of `setTimeout` here. This should mock `setTimeout` with a function that returns a mock timeout object.

Reply:
Fake Timers is useful here because it will take care of that mocking for you. 

https://github.com/sinonjs/fake-timers#api-reference